tools/querytee/proxy_metrics.go

```diff
@@ -24,18 +25,18 @@ type ProxyMetrics struct {
 func NewProxyMetrics(registerer prometheus.Registerer) *ProxyMetrics {
 	m := &ProxyMetrics{
 		requestDuration: promauto.With(registerer).NewHistogramVec(prometheus.HistogramOpts{
-			Namespace: "cortex_querytee",
-			Name:      "request_duration_seconds",
+			Namespace: queryTeeMetricsNamespace,
+			Name:      "backend_request_duration_seconds",
```

Comment:
This is a breaking change, but there is a conflict with the weaveworks Server metric: https://github.com/weaveworks/common/blob/master/server/server.go#L275-L281

An alternative non-breaking change would be to use a different namespace for the weavework server. Wdyt?

Reply:
We can make this breaking change. Please mention it as a `[CHANGE]` in the changelog (add a section `### Query-tee`) and update the metric name referenced in the doc page `docs/sources/operators-guide/tools/query-tee.md`